api/src/mailer.ts

```diff
@@ -59,7 +59,15 @@ export default function getMailer(): Transporter {
 				host: env.EMAIL_MAILGUN_HOST || 'api.mailgun.net',
 			}) as any
 		);
-	} else {
+	} else if (transportName === 'sendgrid') {
+		const sg = require('nodemailer-sendgrid');
+		const transport = nodemailer.createTransport(
```

Comment:
Lil bug here. This should be `transporter = ` instead of `const transport =`, otherwise this transporter won't actually be used by the mailer instance